lib/index.js

```diff
@@ -1,8 +1,17 @@
 const express = require('express')
 const bodyParser = require('body-parser')
+var dbLib = require('../lib/database')
+var dbConfig = require('../../db-config.js')
```

Comment:
Die Mongo-URL beinhaltet auch den DB-benutzer und Kennwort. Daher habe ich das in einen anderen Ordner gemacht, damit man nicht unsere Zugangsdaten von den Deployment-Servern sehen kann.

Ausserdem kann sich so jeder seine eigene Konfig machen, ohne dass die immer überschrieben wird. Das ist aber nur ein Nebengrund =)